blog/resources/debug/cache/modulefederation.config.js

```diff
@@ -0,0 +1,28 @@
+const { dependencies } = require('./package.json');
```

Comment:
Should the cache be in git?

Reply:
it's not cache, it's sources for cache panel